pkg/controller/cassandracluster/cassandra_util.go

```diff
@@ -34,6 +34,10 @@ func (rcc *ReconcileCassandraCluster) thereIsPodDisruption() bool {
 	return rcc.storedPdb.Status.PodDisruptionsAllowed == 0
 }
 
+func (rcc *ReconcileCassandraCluster) weAllowMoreThan1PodDisruption() bool {
```

Comment:
Let's rename that one to `allowPodDisruption`

Reply:
go pout allowMoreThan1PodDisruption (cf le test)